images.v2.yaml

```diff
@@ -0,0 +1,262 @@
+    - ufoym/deepo
+    - ufoym/deepo:all-py36
+    - ufoym/deepo:all-py36-cpu
+    - valohai/darknet:62b781a-cuda8.0-cudnn5-devel-ubuntu16.04
+    - valohai/darknet:b61bcf5-cuda8.0-cudnn5-devel-ubuntu16.04
+    - valohai/hauki:tensorflow1.4.1-python3.5-cuda8.0-cudnn6.0-devel-ubuntu16.04-joonas
+    - valohai/hauki:tensorflow1.4.1-python3.5-opencv3.4-dlib19.8-cuda8.0-cudnn6.0-devel-ubuntu16.04
+    - valohai/keras:2.0.0-tensorflow1.0.1-python3.6-cuda8.0-cudnn5-devel-ubuntu16.04
+    - valohai/keras:2.0.0-theano0.8.2-python3.6-cuda8.0-cudnn5-devel-ubuntu16.04
+    - valohai/keras:2.0.0-theano0.9.0rc4-python3.6-cuda8.0-cudnn5-devel-ubuntu16.04
+    - valohai/keras:2.1.3-tensorflow1.4.0-python3.5-cuda8.0-cudnn6-devel-ubuntu14.04
+    - valohai/keras:2.1.4-theano1.0.1-python3.6-cuda9.0-cudnn7-devel-ubuntu16.04
+
+  aliases:
+    busybox:1.28.3:
+      - busybox:1
+      - busybox:1.28
+    python:2.7.14-alpine3.7:
+      - python:2-alpine3.7
+      - python:2.7-alpine3.7
+    python:2.7.14-jessie:
+      - python:2
+      - python:2.7
+      - python:2.7.14
+    python:3.6.5-alpine3.7:
+      - python:3-alpine3.7
+      - python:3.6-alpine3.7
+      - python:alpine3.7
+    python:3.6.5-jessie:
+      - python:3
+      - python:3.6
+      - python:3.6.5
+    tensorflow/tensorflow:1.5.0-devel-gpu:
+      - gcr.io/tensorflow/tensorflow:1.5.0-devel-gpu
+    tensorflow/tensorflow:1.5.0-devel-gpu-py3:
+      - gcr.io/tensorflow/tensorflow:1.5.0-devel-gpu-py3
+    tensorflow/tensorflow:1.6.0-devel:
+      - gcr.io/tensorflow/tensorflow:1.6.0-devel
+    tensorflow/tensorflow:1.6.0-devel-gpu:
+      - gcr.io/tensorflow/tensorflow:1.6.0-devel-gpu
+    tensorflow/tensorflow:1.6.0-devel-gpu-py3:
+      - gcr.io/tensorflow/tensorflow:1.6.0-devel-gpu-py3
+    tensorflow/tensorflow:1.6.0-devel-py3:
+      - gcr.io/tensorflow/tensorflow:1.6.0-devel-py3
+    ufoym/deepo:all-py27:
+      - ufoym/deepo:py27
+    ufoym/deepo:all-py27-cpu:
+      - ufoym/deepo:py27-cpu
+    ufoym/deepo:all-py36:
+      - ufoym/deepo:all
+      - ufoym/deepo:py36
+    ufoym/deepo:all-py36-cpu:
+      - ufoym/deepo:all-cpu
+      - ufoym/deepo:cpu
+      - ufoym/deepo:py36-cpu
+
+  suggestions:
+    - ufoym/deepo:all-py36
+    - ufoym/deepo:all-py36-cpu
+    - ufoym/deepo:keras-py36-cu90
+    - pytorch/pytorch:1.5.1-cuda10.1-cudnn7-runtime
+    - pytorch/pytorch:1.4-cuda10.1-cudnn7-devel
+    - tensorflow/tensorflow:1.13.1-gpu-py3
+    - tensorflow/tensorflow:1.13.1-py3
+    - tensorflow/tensorflow:1.15.0-gpu-py3
+    - tensorflow/tensorflow:1.15.2-gpu-py3
+    - tensorflow/tensorflow:2.0.0-gpu-py3
+    - tensorflow/tensorflow:2.0.1-gpu-py3
+    - tensorflow/tensorflow:2.2.0-py3
+    - tensorflow/tensorflow:2.2.0-gpu-py3
+    - valohai/keras:2.0.0-tensorflow1.0.1-python3.6-cuda8.0-cudnn5-devel-ubuntu16.04
+    - valohai/keras:2.0.0-theano0.9.0rc4-python3.6-cuda8.0-cudnn5-devel-ubuntu16.04
+    - valohai/darknet:b61bcf5-cuda8.0-cudnn5-devel-ubuntu16.04
```

Comment:
Can we also get rid of this list in v2? (Aren't these the `isRecommended: True` ones?)

Should the aliases also be moved under each `details` entry? 😇 

How about also getting rid of `cached` in v2? It's out of date anyway.

Reply:
Uh, I'm not sure if recommended === suggested? Suggested means they exist, like, everything in the details list. Then, some of them are especially recommended (noted with a fancy badge).

Reply:
Well, should we maybe add `isSuggested: true` then too? Just that now that we are overhauling these, maybe all that data could be in the single `details` dict?

Reply:
But yes, aliases can move under details and the complete list of details can be used as "suggestions" with info?